sender/debug.go

```diff
@@ -54,8 +54,7 @@ func (db *Debug) Send(c *skogul.Container) error {
 	}
 	b, err := e.Encode(c)
 	if err != nil {
-		debugLog.WithError(err).Panic("Unable to encode for debug output")
-		return err
+		return fmt.Errorf("encoding failed: %w", err)
```

Comment:
If the `Panic()` call here is dropped, this changes how this works (it'll no longer panic). But that's probably fine..? It's also the debug sender, which probably isn't used heavily in regular setups.

Reply:
Good catch... I can't imagine we WANT it to panic, but it certainly changes behavior. I'd say for the better, so leaving this change in.